packages/cohort/src/snapshot_api.rs

```diff
@@ -11,6 +11,7 @@ use crate::state::postgres::database::{Database, SNAPSHOT_SINGLETON_ROW_ID};
 
 pub static SNAPSHOT_UPDATE_QUERY: &str = r#"UPDATE cohort_snapshot SET "version" = ($1)::BIGINT WHERE id = $2 AND "version" < ($1)::BIGINT"#;
 
+#[derive(Debug, Clone)]
```

Comment:
This does not need to be cloned, as we are not creating instances of it :)

Reply:
left over of some experiment I was doing. :) Will remove the entire derive.